README.md

```diff
@@ -9,17 +9,32 @@ There is no "release" yet. The current milestone is "1.5 Stable", which is meant
 The milestone needs testing on all platforms. Please report issues when found.
 
 ## Usage
-Purchase Descent 3 from a reputable source, such as [GOG](https://www.gog.com/game/descent_3_expansion) or [Steam](https://store.steampowered.com/app/273590/Descent_3/), and install it, then replace the main binary with the newly built `Descent3` binary under `${CMAKE_BINARY_DIR}/Descent3/*/Descent3[.exe]`.
-See your platform below:
+1. Make sure that you have a copy of Descent 3. You can purchase a copy of Descent 3 from [GOG](https://www.gog.com/game/descent_3_expansion) or [Steam](https://store.steampowered.com/app/273590/Descent_3/).
+2. Install Descent 3.
 
-#### Windows
-If still using the Descent 3 launcher, copy your `Descent3.exe` binary to your install folder and rename it `main.exe` (back up your old one). Otherwise, drop in `Descent3.exe` and play!
+    - **Note for Steam users:** If you own Descent 3 on Steam, then it’s recommended that you install the Windows version of the game even if you’re running macOS or Linux. [If you do not do this, then cutscenes will not work.](https://github.com/DescentDevelopers/Descent3/issues/240) You can use either [Steam Play](https://help.steampowered.com/en/faqs/view/08F7-5D56-9654-39AF) or [SteamCMD](https://developer.valvesoftware.com/wiki/SteamCMD#Cross-Platform_Installation) to install the Windows version of the game on macOS or Linux.
```

Comment:
We can be more specific here, it's alright.

```
**Note for Steam users:** If you own Descent 3 on Steam, then it’s recommended that you install the Windows version of the game even if you’re running macOS or Linux, otherwise movies will not work due to [current lack of Ogv support](https://github.com/DescentDevelopers/Descent3/issues/240).
```

Reply:
OK. I just pushed a new version that incorporates this suggestion.